frontend/src/components/Dialog/User/DeleteUser.vue

```diff
@@ -32,7 +36,7 @@ async function deleteUser() {
 }
 </script>
 <template>
-  <v-dialog v-model="show" max-width="500px" :scrim="true">
+  <v-dialog v-if="user" v-model="show" max-width="500px" :scrim="true">
```

Comment:
you'll see this in a couple places, it's just to make ts happy and should have no impact on the component